akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -474,6 +472,80 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>standalone</id>
+            <properties>
+                <docker.base.image>eclipse-temurin:21.0.1_12-jre-jammy</docker.base.image>
+            </properties>
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <groupId>io.kalix</groupId>
+                        <artifactId>akka-runtime-core_2.13</artifactId>
+                        <version>${kalix-runtime.version}</version>
+                    </dependency>
+                </dependencies>
+            </dependencyManagement>
+
+            <dependencies>
+                <dependency>
+                    <groupId>io.kalix</groupId>
+                    <artifactId>akka-runtime-core_2.13</artifactId>
```

Comment:
Won't this put all the classes back on the classpath, or would you enable the profile only when building so IDE would not pick it up?

Reply:
I tested with IntelliJ, there it doesn't 👍